lib/zoom/actions/user.rb

```diff
@@ -12,8 +12,11 @@ def user_list(*args)
 
       def user_create(*args)
         params = Zoom::Params.new(Utils.extract_options!(args))
-        params.require(%i[email type first_name last_name password])
-        Utils.parse_response self.class.post('/users', body: { action: 'create', user_info: params }.to_json, query: { access_token: access_token })
+        require_param_keys = %i[action email type]
+        require_param_keys.append(:password) if params[:action] == 'autoCreate'
+        params.require(require_param_keys)
+        params.permit_value(:action, Zoom::Constants::USER_CREATE_TYPES.values)
```

Comment:
Ref Comment on `lib/zoom/contsants.rb` 
```suggestion
        params.permit_value(:action, Zoom::Constants::USER_CREATE_TYPES.keys)
```

Reply:
I fixed. Thanks for your review!